src/parser/jobs/gnb/modules/Cooldowns.tsx

```diff
@@ -25,8 +25,8 @@ ________________________________________________
 Quick Maths, 2500 * GCD # = Offset, take the higher number for the offset.
 */
 
-const FIRST_USE_OFFSET_BLOODFEST = 5000
-const ALLOWED_BLOODFEST_HOLDTIME = 10000 // Bloodfest is allowed to be held in order to delay it back under buffs, this should be a maximum of 4 GCDs, 2 ahead of NM, 2 after NM cast to dump ammo.
+const FIRST_USE_OFFSET_BLOODFEST = 2500
+let ALLOWED_BLOODFEST_HOLDTIME = 10000 // Bloodfest is allowed to be held in order to delay it back under buffs, this should be a maximum of 4 GCDs, 2 ahead of NM, 2 after NM cast to dump ammo.
```

Comment:
I feel like in this case it'd be better to go the ALLOWED_BLOODFEST_HOLDTIME_[BEFORE/AFTER]_[PATCH] route, since we really shouldn't be having re-assignable variables outside of classes like this.